tests/core/test_config.py

```diff
@@ -20,52 +20,18 @@ def __init__(self, database_transaction: DatabaseTransactionFixture):
         self.Conf = MockConfiguration
         self.Conf.instance = dict()
         self.root_dir = os.path.join(os.path.split(__file__)[0], "..", "..")
-        self.VERSION_FILENAME = os.path.join(self.root_dir, self.Conf.VERSION_FILENAME)
         self.transaction = database_transaction
 
-    def close(self):
-        if os.path.exists(self.VERSION_FILENAME):
-            os.remove(self.VERSION_FILENAME)
-
 
 @pytest.fixture()
 def configuration_test_fixture(
     db: DatabaseTransactionFixture,
 ) -> Iterable[ConfigurationTestFixture]:
     fix = ConfigurationTestFixture(db)
-    yield fix
-    fix.close()
+    return fix
```

Comment:
Minor: We could collapse lines 30 & 31 here to:
```
return ConfigurationTestFixture(db)
```

Reply:
Done